apps/core/src/contexts/HiddenAssetsProvider.tsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
// Copyright (c) 2024 IOTA Stiftung
// SPDX-License-Identifier: Apache-2.0
```

Reply:
it's not a new file; it has just been moved to core

Reply:
fair!